mixer/pkg/runtime/config/config_test.go

```diff
@@ -2440,38 +2430,6 @@ Attributes:
 `,
 		wantErr: "",
 	},
-	{
-		Name: "add handler - bad param type",
-		Events1: []*store.Event{updateEvent("a1.adapter.default", &adapter_model.Info{
```

Comment:
Are these tests replaced by something else, or just removed?

Reply:
The original code path for these is gone. That's why I removed it. However, there is still some similar (unlikely) downstream error paths, which I am not sure is captured in other tests, so I resurrected them to cover those.